prompts/deepset-conversational-agent.md

```diff
@@ -0,0 +1,29 @@
+This prompt is intended to be used with a Haystack `Agent`, specifically, a [`ConversationalAgent`](https://docs.haystack.deepset.ai/docs/agent#conversational-agent).
+
+To learn how to use this prompt and the `ConversationalAgent`, check out our [Building a Conversational Chat App Tutorial](https://haystack.deepset.ai/tutorials/24_building_chat_app).
+
+By default, the `ConversationalAgent` would use this prompt when it's provided with tools. If you want to more flexibility, below is an example of how you might use this prompt with the `Agent`.
```

Comment:
```suggestion
By default, the `ConversationalAgent` would use this prompt when it's initialized with tools. If you want more flexibility, below is an example of how you might use this prompt with the `Agent`.
```

Reply:
I welcome any other suggestion than "initialized", it's just that "provided" is a little bit vague here.
Another option:
```suggestion
By default, the `ConversationalAgent` would use this prompt when it's used with tools. If you want more flexibility, below is an example of how you might use this prompt with the `Agent`.
```

Reply:
I think 'initialized' is a great option here, I'll go with that 👍